### DISCUSSION TITLE
[3.x] Add another component/view in the wizard body

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#7855] [Help] [answered]
Is it possible to add another component/view in the wizard that is not a filament form component?
![image](https://github.com/filamentphp/filament/assets/29732308/9f0434fb-8199-4187-ac07-38c045cac1cb)

**Comment by pboivin** [comment, marked as answer]
I think this can be done with a Grid (e.g. 3 columns). The left side could be a custom `View` component and the right side could be your regular form fields.

**Reply by ArtMin96** [reply]
Thanks @pboivin, I will try.
